packages/api-reference/src/helpers/convertRequestToHarRequest.ts

```diff
@@ -0,0 +1,103 @@
+import type { HarRequest } from 'httpsnippet-lite'
+
+/** Takes in a regular request object and returns a HAR request */
```

Comment:
did claude do this right @hanspagel ?

Reply:
Looks good. We could/should? import the type from `@scalar/types/external`.

Reply:
oh I got it now 🫡